src/robotlibcore.py

```diff
@@ -142,6 +135,37 @@ def get_keyword_documentation(self, name):
             doc = '{}\n\n{}'.format(doc, tags) if doc else tags
         return doc
 
+    def get_keyword_types(self, keyword_name):
+        method = self.keywords.get(keyword_name)
+        if not method:
+            raise ValueError('%s is not keyword.' % keyword_name)
+        robot_types = self._get_robot_types(method)
+        if robot_types is not False:
+            if robot_types is None:
+                return {}
```

Comment:
Should return `None`, not `{}`, if `robot_types is None`.  The difference is that if explicit types are `None`, Robot shouldn't do automatic argument conversion at all, not even based on default values. At the moment RF doesn't actually work like that but I need to fix it on Robot side.

As already commented above, the `if robot_types is not False` condition can be removed altogether.

Reply:
The aforementioned issue on Robot side has been fixed.